src/envs/behavior_options.py

```diff
@@ -0,0 +1,976 @@
+"""Hardcoded options for BehaviorEnv."""
```

Comment:
general comment: I am truly impressed that you figured all of this out, and very grateful that I didn't have to myself 😅